website/pages/community/index.jsx

```diff
@@ -18,27 +18,27 @@ export default function CommunityPage() {
           {
             header: 'Community Forum',
             body:
-              '[Consul Community Forum](https://discuss.hashicorp.com/c/consul)',
+              '<a href="https://discuss.hashicorp.com/c/consul">Consul Community Forum</a>',
```

Comment:
`vertical-text-block-list` no longer processes markdown internally, so we need to pass it HTML instead. This could be done automatically in `getStaticProps` if we feel it'd be worth it for authoring convenience.